source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst

```diff
@@ -4,44 +4,68 @@ Exporting data
 .. image:: images/export.png
    :alt: Export a dataset
 
-Opendatasoft datasets can be downloaded in various formats. Standard formats are always available for any dataset. Other more specific formats can be available, but they depend on the nature of the data (that is, if the dataset contains geographical coordinates or not).
+Opendatasoft datasets can be downloaded in various formats.
+Flat file formats are always available for any dataset.
+More specific formats can be available, depending on whether the dataset contains geographical coordinates or not.
 
-In the Export tab of a dataset, the following formats can be available:
+1. Go to the **Export** tab of the dataset.
+2. Choose the file format to download.
+3. Perform one of the following actions:
 
-- CSV
-- JSON
-- Excel
-- GeoJSON
-- Shapefile
-- KML
+   * Click **Whole dataset** next to the desired format.
+   * If the dataset is filtered and you want to export the filtered data only, click **Only the selected records** next to the desired format.
 
 .. admonition:: Note
    :class: note
 
-   It is possible to download the dataset in other formats using the `Opendatasoft Search API <https://help.opendatasoft.com/apis/ods-search-v2/#exporting-records>`_.
+   - You can use the parameter :code:`use_labels_for_header=true` in the exports links to use the labels instead of the technical identifiers of the fields as a header (not available in the Opendatasoft Search API v2).
+   - You can also export datasets in other formats using the `Opendatasoft Search API v2 <https://help.opendatasoft.com/apis/ods-search-v2/#exporting-records>`_.  
 
-.. admonition:: Note
-   :class: note
+Available export formats
+------------------------
 
-   There are 2 limitations in Shapefile exports.
+In the Export tab of a dataset, the following formats can be available:
 
-   - The export cannot contain more than 50,000 records.
-   - The export cannot contain more than one geographical shape type (e.g. point, polygon, linestring etc.). Only the first type of the first non-empty record will be kept in the export.
+Flat file formats
+~~~~~~~~~~~~~~~~~
 
-1. Go to the Export tab of the dataset.
-2. Choose the file format to download.
-3. Click on "Whole dataset" next to the chosen format.
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * * Format
+     * Notes
+   * * CSV
+     * CSV uses semicolon (;) as a separator. 
+   * * JSON
+     * 
+   * * Excel
+     * 
+
+Geographic file formats
+~~~~~~~~~~~~~~~~~~~~~~~
 
 .. admonition:: Important
-   :class: important
+   :class: Important
 
-   If the dataset is filtered, it is possible to only download the filtered data. To do so, click on "Only the selected records" instead of "Whole dataset".
+   The geographical coordinates of a dataset are expressed in WGS84 by default. When available, you can select additional geographical projections from the drop-down displayed above the export formats.
```

Comment:
On the "when available", I would add something like "when made available by the data producer" to show the difference between "geo formats are available automatically when geographical data is present" and "geo projections are manually chosen by the data producer" (and the re-users could ask the producers about their preferred projection if it’s really important).

Reply:
Fixed in 9da1777ab424022b02d96e9ff6512efdd7579538